src/app/src/components/annotations/annotator.tsx

```diff
@@ -649,8 +652,9 @@ export default class Annotator extends Component<
       // eslint-disable-next-line no-await-in-loop
       await this.getInference(asset, true);
       if (
-        this.state.inferenceOptions.bulkAnalysisStatus === "both" ||
-        this.state.inferenceOptions.bulkAnalysisStatus === asset.type
+        (this.state.inferenceOptions.bulkAnalysisStatus === "both" ||
+          this.state.inferenceOptions.bulkAnalysisStatus === asset.type) &&
```

Comment:
I think this becomes rather unnecessary given that you're already filtering on the top level

Reply:
got it!